generate-release/src/release_notes.rs

```diff
@@ -107,3 +147,68 @@ file_name = "{file_name}.md"
         title = title.trim().replace('"', "\\\"")
     )
 }
+
+#[allow(clippy::too_many_arguments)]
+fn generate_and_open_issue(
+    client: &GithubClient,
+    issue_titles: &HashSet<String>,
+    pr: &GithubIssuesResponse,
+    title: &str,
+    authors: &[String],
+    file_path: &Path,
+    milestone: &str,
+    dry_run: bool,
+) {
+    let pr_number = pr.number;
+    let issue_title = format!("Write release notes for PR #{pr_number}: {title}");
+
+    // Check if this issue already exists
+    // If it does, we don't want to spam the repo with duplicate issues
+    if issue_titles.contains(&issue_title) {
+        return;
+    }
+
+    let pr_url = format!("https://github.com/bevyengine/bevy/pull/{pr_number}",);
+    let file_path = file_path.to_string_lossy();
+
+    let authors = authors
+        .iter()
+        .map(|author| format!("{}", author))
+        .collect::<Vec<_>>()
+        .join(", ");
+
+    let issue_body = format!(
+        "{pr_url} needs release notes for the upcoming Bevy release!
+        Please reply below if you'd like to volunteer do so, whether or not you're the author of the PR.
+        
+        Release notes should:
+        
+        1. Clearly motivate the change.
+        2. Be written in a way that is understandable by the average Bevy user: some programming background and a general understanding of games.
+        3. Show off the coolest features of the PR. Screenshots are awesome, but elegant APIs are also welcome!
+        4. If this was a perf-centric PR, quantify the performance improvements. Graphs and statistics work well for this.
+
+        We can help you revise the release notes: a rough draft alone is incredibly useful :)
+        Your expertise is invaluable for contextualizing the changes; we'll work with you to bring the technical writing up to par.
+
+        To submit your release notes, modify `{file_path}.md` and submit a PR.
```

Comment:
```suggestion
        To submit your release notes, modify `{file_path}` and submit a PR.
```

`.md` is already in this filename, so it gets duplicated with this!